Branch.podspec

```diff
@@ -21,7 +21,15 @@ Use the Branch SDK (branch.io) to create and power the links that point back to
   s.platform     = :ios, '6.0'
   s.requires_arc = true
 
-  s.source_files = "Branch-SDK/Branch-SDK/*.{h,m}", "Branch-SDK/Fabric/*.h", "Branch-SDK/Branch-SDK/Requests/*.{h,m}"
+  s.subspec 'Core' do |core|
```

Comment:
Won't this require us to change all integrated pod files to `pod 'Branch/Core'` or will `pod Branch` still work?